asyncapi-cli/src/main/resources/listener.bal

```diff
@@ -7,11 +7,11 @@ public class Listener {
     private http:Listener httpListener;
     private DispatcherService dispatcherService;
 
-    public function init(int|http:Listener listenOn = 8090) returns error? {
-        if listenOn is http:Listener {
-            self.httpListener = listenOn;
+    public function init(int|http:Listener listenTo = 8090, *ListenerConfiguration configuration) returns error? {
+        if listenTo is http:Listener {
+            self.httpListener = listenTo;
         } else {
-            self.httpListener = check new (listenOn);
+            self.httpListener = check new (listenTo, configuration);
```

Comment:
Don't we need to check the HTTP version here? If the version is 2.0, should throw an error, I guess?

Reply:
IMO we don't need to as a webhook can work either with HTTP v1.1 or HTTP v2.0. @TharmiganK correct me if I am wrong.

Reply:
Yes, the service will use the HTTP version according to the request received